CHANGES.rst

```diff
@@ -1,3 +1,14 @@
+1.5.10 (April 15, 2020)
+=======================
+Bug-fix release in the 1.5.x series.
+
+This release fixes a bug for phase-difference fieldmaps that are not in RAS+ orientation.
```

Comment:
to emphasize
```suggestion
This release fixes a bug for **phase-difference fieldmaps that are not in RAS+ orientation**.
```

Reply:
```suggestion
This release fixes a bug for **phase-difference fieldmaps that are not in RAS+ orientation**.
When the non-canonical orientation (any other than RAS+) included a reordering of axis
(e.g. ARS+), *fMRIPrep* crashed when the number of voxels of the swapped dimensions did not
match.
Otherwise, the bug introduced a poor masking - oftentimes unnoticeable if the original orientation
was LAS+)
```